applications/FluidDynamicsApplication/custom_processes/shock_capturing_physics_based_process.h

```diff
@@ -194,6 +189,10 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) ShockCapturingPhysicsBasedProcess :
     bool mShockSensor;
     bool mShearSensor;
     bool mThermalSensor;
+    double mK_beta_constant  = 0.0;
+    double mPr_beta_constant = 0.0;
+    double mK_kappa_constant = 0.0;
+    double mK_mu_constant    = 0.0;
```

Comment:
```suggestion
    double mArtBulkViscosityConstant  = 0.0;
    double mFarFieldPrandtl = 0.0;
    double mArtConductivityConstant = 0.0;
    double mArtiDynViscosityConstant    = 0.0;
```
Note 2 things about the variables naming:
- First is our convention, which you can found in the [Kratos style guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide).
- Second is that it is a good practice to give self-descriptive names to vars. In here, `K` and `constant` are redundant and beta, kappa and mu are not self-descriptive so I suggest changing them to the physical magnitudes they refer to.